tests/testSpaces/testSetup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+source "$EUPS_DIR/bin/setups.sh"
+
+cd "space path"
+
+setup -k -r .
+if [ $(eups list 2>&1 | grep -c Problem) != "0" ]; then
+  echo Problem detected in eups list
+  exit 1
+fi
+
+unsetup -r .
+unsetup eups
```

Comment:
Move this script (and any associated data) to the `testEupsIntegration` suite (just prepend/append setup.sh/teardown.sh to it). To make it more self-contained, you could even generate the contents of `space path` from within the script (instead of having `space path.orig`).